### PR TITLE
Add auto archive batch configuration

### PR DESCRIPTION
**Pull request by danielyag**

### CODE REVIEW

File: configurations/batch/batch.ini.template [hunk 1]
```diff
@@ -68,6 +68,7 @@ enabledWorkers.KAsyncCopyCaptions					= 1
 enabledWorkers.KAsyncCopyCuePoints                   = 1
 enabledWorkers.KSyncReachCreditTaskRunner	    = 0
 enabledWorkers.KAsyncExportCsv                      = 0
+enabledWorkers.KAsyncLiveEntryArchive               = 1
```
Comment (yossipapi) [inline configurations/batch/batch.ini.template]: Any reason this was added to the template? Do we really need to run this batch on all prod machines?

Comment (danielyag) [reply configurations/batch/batch.ini.template]: We added it to work as LiveToVOD

File: configurations/batch/batch.ini.template [hunk 2]
```diff
@@ -800,6 +801,12 @@ scriptPath                                          = batches/ServerNodeMonitor/
 params.serverNodeTTL                                = 120
 params.typesToMonitor                               = 1
 
+[KAsyncLiveEntryArchive : JobHandlerWorker]
+id                                                  = 680
```
Comment (yossipapi) [inline configurations/batch/batch.ini.template]: Please note this id is already taken by KAsyncCopyCaptions
[KAsyncCopyCaptions : JobHandlerWorker]
id                                    = 680

Please double check yourself before attaching an id to a worker

File: configurations/batch/batch.ini.template (hunk 1 above)
Comment (yossipapi) [reply configurations/batch/batch.ini.template]: So why do you need another worker if this is part of the liveTovod process?
Am I missing something?

Comment (danielyag) [reply configurations/batch/batch.ini.template]: It has two scenarios: 
1) Work automatically after the live to VOD finished
2) Work when a proper API call was triggered

**Pull request merged**